core/cas-server-core-web/src/main/java/org/apereo/cas/config/CasCoreWebConfiguration.java

```diff
@@ -30,6 +33,24 @@
     @Autowired
     private CasConfigurationProperties casProperties;
 
+    /**
+     * Get messages from one or more property files containing values
+     * that should be exposed to Thyme templates but don't need i18n.
+     * This bean is Autowired into {@link messageSource()}.
+     * @return PropertiesFactoryBean containing all common (non-i18n) messages
+     */
+    @Bean
+    public PropertiesFactoryBean casCommonMessages() {
```

Comment:
I realize it's not necessary for this requirement to use a PropertiesFactoryBean instead of just a regular Properties object, but I don't think using a PropertiesFactoryBean is that abnormal. If Spring sees a bean implementing FactoryBean interface then it just calls 'Properties getObject()' and the returned Properties object is the bean, there is no proxy involved. This is just one bean being autowired into another bean. Below is an example where multiple property files are merged into one by the PropertiesFactoryBean and then implementors could put entries in a custom-common-messages.properties file that would be available to ThymeLeaf templates. 

```
    @Bean
    public PropertiesFactoryBean casCommonMessages() {
        final PropertiesFactoryBean properties = new PropertiesFactoryBean();
        final Resource[] propertyResources = new Resource[] {
            new ClassPathResource("/static-uris.properties"),
            new ClassPathResource("/custom-common-messages.properties"),
        };
        properties.setLocations(propertyResources);
        properties.setSingleton(true);
        properties.setIgnoreResourceNotFound(true);
        return properties;
    }
```

I will update the pull request to create and load a regular Properties object and set that, but it seems odd to have "static-uris.properites" as THE common messages property file. Maybe I should rename it to common-messages.properties or non-i18n-messages.properties? 

My secret goal in using PropertiesFactoryBean is avoiding the checked IOException but I guess I can use UncheckedIOException to re-throw it, or maybe I should log it... 
```
        try {
        	Properties p = new Properties();
		p.load(new ClassPathResource("/static-uris.properties").getInputStream());
		bean.setCommonMessages(p);
	} catch (IOException e) {
		throw new UncheckedIOException(e);
	}
```